### DISCUSSION TITLE
File ID syntax in EE7 has some workflow gaps when managing files using the File Manager

### DISCUSSION THREAD

**Discussion by PaulBaileyAtChan** [ExpressionEngine/ExpressionEngine#4471] [Feature Requests]
### Suggest an idea for ExpressionEngine

The new file ID syntax introduced in EE7 enables some nice features, but it feels like the workflow hasn't entirely been thought through.

This is especially the case when managing files in `textarea` fields, which might contain arbitrary text and image content. Specifically, it's not easy to relate the file ID syntax to the information in the File Manager. Prior to EE7, the inline file syntax included the full filename; this could then be used to locate a file in the File Manager. In EE7, the only information currently provided in content which identifies a file is its ID.

But it's not currently possible to search the File Manager for a file's ID, so locating a file is actually harder than before, especially in a set-up which contains a lot of files. Filenames can be grabbed from a site's front-end, but this is clumsy. Relating this to a workflow example, removing a file from an entry is easy enough, but it's not at all clear how to delete the same file — given its ID — from the File Manager.

**As a minimum, it should be possible to easily locate a file in the File Manager, given its ID.**

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4476
